test/RB/rb_generate_data.py

```diff
@@ -0,0 +1,121 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Code for generating data for the RB experiment for testing data analysis.
+"""
+
+import os
+import sys
+import json
+from qiskit.providers.aer import QasmSimulator
+from qiskit.providers.aer.noise import NoiseModel
+from qiskit.providers.aer.noise.errors.standard_errors import depolarizing_error
+import qiskit_experiments as qe
+
+
+def create_depolarizing_noise_model():
+    """
+    create noise model of depolarizing error
+    Returns:
+        NoiseModel: depolarizing error noise model
+    """
+    # the error parameters were took from ibmq_manila on 17 june 2021
+    p1q = 0.002257
+    p2q = 0.006827
+    noise_model = NoiseModel()
+    noise_model.add_all_qubit_quantum_error(depolarizing_error(p1q, 1), "x")
+    noise_model.add_all_qubit_quantum_error(depolarizing_error(p1q, 1), "sx")
+    noise_model.add_all_qubit_quantum_error(depolarizing_error(p2q, 2), "cx")
+    return noise_model
+
+
+def standard_rb_exp_data_gen(dir_name: str):
+    """
+    Encapsulation for different experiments attributes which in turn execute.
+    The data and analysis is saved to json file via "_generate_rb_fitter_data" function.
+    Args:
+        dir_name(str): The directory which the program will save the data and anaysis.
+    """
+    rb_exp_name = ["rb_standard_1qubit", "rb_standard_2qubits"]
+    experiments_attributes = [
+        {
+            "qubits": [0],
+            "lengths": list(range(1, 200, 20)),
```

Comment:
I think that length of 200 is not enough for 1-qubit RB, since you won't see it decreasing closer to 0.5.
In the tutorial they use 500:
https://github.com/Qiskit/qiskit-experiments/blob/main/docs/tutorials/rb_example.ipynb